src/main/resources/l10n/JabRef_es.properties

```diff
@@ -2285,21 +2290,11 @@ Don't\ share=No compartir
 Share\ anonymous\ statistics=Compartir estadísticas anónimas
 Telemetry\:\ Help\ make\ JabRef\ better=Telemetría\: Ayude a mejorar JabRef
 To\ improve\ the\ user\ experience,\ we\ would\ like\ to\ collect\ anonymous\ statistics\ on\ the\ features\ you\ use.\ We\ will\ only\ record\ what\ features\ you\ access\ and\ how\ often\ you\ do\ it.\ We\ will\ neither\ collect\ any\ personal\ data\ nor\ the\ content\ of\ bibliographic\ items.\ If\ you\ choose\ to\ allow\ data\ collection,\ you\ can\ later\ disable\ it\ via\ Options\ ->\ Preferences\ ->\ General.=Para mejorar la experiencia de usuario, nos gustaría recopilar estadísitcas anónimas sobre las funcionalidades de JabRef que usa. Sólo se registrará qué funcionalidades emplea y con qué frecuencia. No se recopilará ningún dato personal ni el contenido de los elementos bibliográficos. Si decide permitir la recopilación de datos, podrá deshabilitarlo posteriormente en Opciones -> Preferencias ->General
-Names\ are\ not\ in\ the\ standard\ %0\ format.=Los nombres no tienen el formato estándar
-
-Delete\ the\ selected\ file\ permanently\ from\ disk,\ or\ just\ remove\ the\ file\ from\ the\ entry?\ Pressing\ Delete\ will\ delete\ the\ file\ permanently\ from\ disk.=¿Borrar el archivo seleccionado del disco duro o solamente quitarlo de la entrada? Presionar el botón de eliminara, borrará el archivo del disco duro permanentemente.
-Delete\ '%0'=Borrar archivo '%0'
-Delete\ from\ disk=Borrar del disco duro
-Remove\ from\ entry=Quitar de la entrada
```

Comment:
@koppor Why does Crowdin revert the changes of https://github.com/JabRef/jabref/pull/3833 ?